src/lib/http-requests.ts

```diff
@@ -36,13 +35,21 @@ class HttpRequests {
     params,
     body,
     config,
+    contentType,
   }: {
     method: string
     url: string
     params?: { [key: string]: any }
     body?: any
     config?: Partial<Request>
+    contentType?: string
   }) {
+    let headers
+    if (contentType !== undefined) {
```

Comment:
Since you are providing a default value in the other methods `post` `put` .. `contentType` we probably never be undefined and thus will always override the `contentType` present in `this.headers`.